code/modules/mob/living/carbon/alien/diona/diona_nymph.dm

```diff
@@ -172,7 +173,7 @@
 
 	icon_state = lowertext(species.name)
 	species.handle_post_spawn(src)
-	maxHealth = species.total_health
+	health = 33.3
```

Comment:
why, I am sure this is already done in mob init

Reply:
done